src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IInput {
+  type: string;
+  name: string;
+  value?: string | number;
+  ref?: React.Ref<HTMLInputElement>;
+  placeholder?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  variant?: string;
```

Comment:
here should be check that estimates accurate words ( like  color?: '' | 'default' | 'green' | 'red',)
becasue below you are checking for 'search' 
How should I know what to pass to Input variant ( it could be any random word